config/application_config.go

```diff
@@ -40,7 +37,7 @@ func (*ApplicationConfig) Prefix() string {
 	return constant.DUBBO + ".application."
 }
 
-// Id ...
+// ID ...
```

Comment:
if u do not wanna add comment here, pls add `// nolint`.

Reply:
ok. It is keep the same as before.